EIPS/eip-2936.md

```diff
@@ -0,0 +1,66 @@
+---
+eip: 2936
+title: EXTCLEAR Opcode For SELFDESTRUCTed contracts
+author: William Morriss (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/eip-2936-extclear-for-selfdestruct/4569
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-03
+replaces: 2751
+---
+
+`EXTCLEAR` Opcode For `SELFDESTRUCT`ed contracts
+
+## Simple Summary
+Enable new opcode to clear storage for `SELFDESTRUCTED`ed contracts.
+
+## Abstract
+`SELFDESTRUCT` complexity can be reduced by deferring state cleanup.
```

Comment:
The abstract is the place to put a human readable and terse version of the specification.  This type of statement is more appropriate in the Motivation section.  For this section, I recommend something like:
```suggestion
Changes `SELFDESTRUCT` (`0xff') to not clear any storage and adds a new `EXTCLEAR` (`0x5c`) opcode that will clear a specific storage slot for a contract that has previously been self destructed.
```